src/edu/csus/ecs/pc2/ui/SubmitRunPane.java

```diff
@@ -481,7 +481,37 @@ protected void testOrSubmitRun(boolean submitTheRun) {
             return;
         }
         
-        if (additionalFilesMCLB.getRowCount() > 0){
+        if (additionalFilesMCLB.getRowCount() > 0) {
+
+            int l = filename.lastIndexOf('/');
```

Comment:
Instead of testing for both / and \, you can use File.separator or File.separatorChar

Reply:
Updated and tested for Windows, need to be tested for Linux and Mac